### PR TITLE
<DOCS>: Updated missing change log entries

### PR DESCRIPTION
**Pull request by SEAJang**
Reason for change:
Change log was missing entries

Resolution:
Update the documentation

Effect(s) of change:
Updated documentation

# Pull Request Key Information

## Reason for change

Changelog was not updated previously. 

## What was changed

Change log was updated

## Intended Purpose

Update missing change log entries

## Level of version change required

None.

## Testing Completed

None needed

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -1,5 +1,17 @@
 # Change Log
 
+## 2023-07 - Version 2.3.0
+
+1. **Added functionality for initializing pre-simulation leaks** Users can now utilize n_init_days and n_init_leaks to initialize pre-simulation start leaks.
```
Comment (ThomasGalesloot) [inline changelog.md:5]: A second sentence adding some more details here might be helpful to users.

File: changelog.md [hunk 2]
```diff
@@ -1,5 +1,17 @@
...
+
+## 2023-07 - Version 2.2.1
+
+1. **Cost bug fix**
```
Comment (ThomasGalesloot) [inline changelog.md:9]: Can we explain what the bug was

File: changelog.md (hunk 1 above)
Comment (SEAJang) [reply changelog.md:5]: My thoughts are that the change log should be brief, key points. If users wanted more details, they should go look at the USER_MANUAL.md to find out more.

File: changelog.md (hunk 2 above)
Comment (SEAJang) [reply changelog.md]: Done.

**Review by ThomasGalesloot — APPROVED** [review]
Looks great now!

**Pull request merged**